!LibuCLib/h/uCLib

```diff
@@ -0,0 +1,26 @@
+#ifndef __ZFP_uCLIB_H
+#define __ZFP_uCLIB_H
+
+#define  UNUSED(x)              (x = x)
+
+#define OS_WriteC    0x00
+#define OS_WriteS    0x01
+#define XOS_WriteS   0x20001
+#define OS_Write0    0x02
+#define XOS_Write0   0x20002
+#define OS_NewLine   0x03
+#define XOS_NewLine  0x20003
+#define OS_Exit      0x11
+#define XOS_Exit     0x20011
```

Comment:
good point, I'll import them from swi.h, forgot about that. Will check if it has any macro or anything that may cause problems when building (just in case). thx!

Reply:
Had a look at SWIs/h but the format doesn't seems to be useful:
```#define TubeSWI   (0x1004)```

Not sure it's the right thing, I need to test this....

Reply:
That's SWIs.h built from HdrSrc. swis.h is provided by RISC_OSLib (autogenerated by the looks of it), I expect you'll find it in your CLib:

Reply:
k, found it and added it, however swis/h in CLib: includes kernel/h which declare (as expected) some _kernel* functions, so need to test is this works or if I may need to build a separate version of swis/h